### ISSUE TITLE
Chore: Single File App tests aren't currently being run by CI

### ISSUE THREAD

**Issue by jamescrosswell** [getsentry/sentry-dotnet#2437]
### TLDR;

`test/SingleFileTestApp/SingleFileTestApp.csproj` needs to be added to the `Sentry-CI-Build-*.slnf` files

### Full context

When our builds are run by GitHub Actions, they use the `Sentry-CI-Build-*.slnf` files.

The code to handle Single File Apps is located in `src/Sentry/Sentry.csproj` and the accompanying tests are in `test/Sentry.Tests/Sentry.Tests.csproj`, both of which are included in the `Sentry-CI-Build-*.slnf` files... However the tests for Single File Apps rely on the `test/SingleFileTestApp/SingleFileTestApp.csproj` file having also been built... which will only be the case if it's included in the relevant `Sentry-CI-Build-*.slnf` for the `runner.os`.

See [build.yml](https://github.com/getsentry/sentry-dotnet/blob/main/.github/workflows/build.yml) for details.

**Comment by jamescrosswell** [comment]
It looks like we did this by accident at some point:
https://github.com/getsentry/sentry-dotnet/blob/0fde2e34300a961fd62fcfd77614ec51b11275e3/Sentry-CI-Build-Windows.slnf#L68

https://github.com/getsentry/sentry-dotnet/blob/0fde2e34300a961fd62fcfd77614ec51b11275e3/Sentry-CI-Build-macOS.slnf#L73

https://github.com/getsentry/sentry-dotnet/blob/0fde2e34300a961fd62fcfd77614ec51b11275e3/Sentry-CI-Build-Linux.slnf#L67
